### PR TITLE
bump new version v-1.9.8-0

### PR DESCRIPTION
**Pull request by mithunjohny**
# PR Details

### Description

_Include a description of the problem to be solved_

## Solution Proposed

_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria

_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence

_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### Sanity Tests

- [ ] You have tested all changes in Popout mode
- [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
- [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y

- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

***Please provide justification if any of the validations has been skipped.***

### CODE REVIEW

**Review by matan-hershberg-nuance — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
```
Comment (matan-hershberg-nuance) [inline CHANGELOG.md:5]: I think normally we keep unreleased as empty, don't we?

**Review by matan-hershberg-nuance — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mithunjohny) [reply CHANGELOG.md:5]: we will push a new branch with the unrelease version in package.json

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
+## [1.9.7]
```
Comment (elopezanaya) [inline CHANGELOG.md:5]: missing unreleased tag on top of new release

**Review by xTEddie — CHANGES_REQUESTED** [review]
Waiting for release of v1.9.7

**Review by elopezanaya — APPROVED** [review]

**Review by xTEddie — APPROVED** [review]
